django_mailbox/models.py

```diff
@@ -336,8 +336,7 @@ def _get_dehydrated_message(self, msg, record):
                 msg.get_payload(decode=True).decode(content_charset)
             except LookupError:
                 logger.warning(
-                    "Unknown encoding %s; interpreting as ASCII!",
-                    content_charset
+                    "Unknown encoding %s; interpreting as ASCII!" % content_charset
```

Comment:
Although it will technically work either way, logging messages *should* *not* be directly interpolated here -- see https://docs.python.org/2/library/logging.html#logging.Logger.debug for more detail.  The formatting of the logging message is intentionally handled deeper inside the logging infrastructure for complicated reasons.